apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerConnection.cs

```diff
@@ -275,9 +255,20 @@ public SpannerConnection(SpannerConnectionStringBuilder connectionStringBuilder)
         /// <paramref name="transactionId"/>.</returns>
         public SpannerTransaction BeginReadOnlyTransaction(TransactionId transactionId)
         {
+            // FIXME: Is this okay? (Users can open asynchronously explicitly if they need to.)
```

Comment:
The difference here is that BeginReadOnlyTransaction *used* to be a completely local operation, without any contact to the outside world or asynchrony involved.

In reality it will almost always act effectively synchronously anyway. I'll remove the FIXME - we can provide more guidance if anyone finds this to be a problem.